### ISSUE TITLE
Removing a Quote from Paragraph Does Not Enable Save Draft Option

### ISSUE THREAD

**Issue by akiashah** [WordPress/gutenberg#59668]
### Description

After adding a paragraph and selecting a quote from it, the "Save Draft" option is enabled as expected. However, if the quote is subsequently removed from the paragraph, the "Save Draft" option does not get enabled again. This behavior suggests that the system does not recognize the removal of a quote as a change that warrants a draft save, potentially leading to loss of progress or confusion.

To generate this bug you need to click save draft after adding it to the paragraph.

### Step-by-step reproduction instructions

Start editing or creating a new block.
Add a paragraph of text.
Select a portion of the text as a quote, triggering the "Save Draft" option to become active/enabled.
Save the draft with the quote included.
Edit the page again and remove the quote from the paragraph.
Observe the behaviour of the "Save Draft" option. it will be not enabled again

### Environment info

**Expected behavior**
It should enable the save draft option while removing the quote from the paragraph.

**Desktop :**

OS: MAC
Browser Chrome
Version 122.0.6261.94
Smartphone (please complete the following information):
NA

**Additional context**
Please create the quote paragraph first and save as draft after this only remove the quote you will find the issue.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by jordesign** [comment]
Hi @akiashah  - I'm not entirely sure what you mean by this part of your instructions:

> Select a portion of the text as a quote,

Could you please elaborate a little?

**Comment by akizwt-dev** [comment]
Hey @jordesign , 
I apologise for the use of too much broad language. 
I'm referring to the text that has been added to the element by anyone in the format of content. 
For your understanding, I am sharing the screenshot link here. https://prnt.sc/9yDW5WwLPeS0

If this is still confusing, I'm happy to create the video for this issue. 

Thanks

**Comment by annezazu** [comment]
Thanks for opening this! It would also be super helpful if you can share what version of WordPress you're using and/or Gutenberg.

**Comment by akiashah** [comment]
@annezazu , the WordPress version is 6.4.3

**Comment by akizwt-dev** [comment]
@annezazu , is there anything else that I may need to add?

**Comment by annezazu** [comment]
No! I just wanted to rule out it being a 6.5 RC related issue :)

**Comment by akiashah** [comment]
Here is the video that I have created quickly, please check and let me know if anyone needs more detail: https://www.awesomescreenshot.com/video/26720615?key=03e0034b62dd3be6292cf62627f76af3

**Comment by carolinan** [comment]
It is not clear to me what this means:

> adding a paragraph and selecting a quote from it

> Select a portion of the text as a quote, 

> remove the quote from the paragraph

> text that has been added to the element by anyone in the format of content.


In what way is this related to the quote block or perhaps the blockquote HTML element?

Did you mean to say that "quote" in this case is any part of a paragraph that is **selected** ?

**Comment by Mamaduka** [comment]
It's unclear what "quote" refers to in this case. Changing paragraph content will enable the "Save" button.

The issue has been without activity for a while. I will close it; it's not actionable without reproduction steps.

@akizwt-dev, if you're still experiencing the issue, please re-open it and include a screencast of the bug.
